cli/devlog.py

```
@@ -71,5 +71,90 @@ def record(story_id: str, action: str, date: datetime | None):
+    
+    # Success path: complete the bounded recording state
+    try:
+        state.complete_bounded_recording(date, story_id, duration, assume_utc=True)
+        click.echo(f"[OK] Bounded recording completed for {story_id} ({duration}s)")
+    except Exception as e:
+        click.echo(f"[ERR] Failed to complete bounded recording for story {story_id}: {e}")
+        # Mark as failed instead of recorded when completion fails
+        try:
+            state.fail_recording(date, story_id, reason=f"State finalization failed: {e}", assume_utc=True)
+            click.echo(f"[INFO] Recording marked as failed for {story_id}")
+        except Exception as cleanup_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)